### ISSUE TITLE
Lambda list 

### ISSUE THREAD

**Issue by Jeanne-jw** [CosmoStat/wf-psf#76]
For the calculation of metrics, there is a small error in lambda list.
In the file[wf-psf](https://github.com/CosmoStat/wf-psf/tree/dummy_main)/[wf_psf](https://github.com/CosmoStat/wf-psf/tree/dummy_main/wf_psf)/[metrics](https://github.com/CosmoStat/wf-psf/tree/dummy_main/wf_psf/metrics)/metrics_interface.py, in the function `evaluate_metrics_mono_rmse()`, lambda_list is set as `np.arange(0.55, 0.9, 0.01)`. However, sometimes the up limit will not be included in `np.arange`. In this case, the last term is 0.89, and $\lambda=0.9\mu m$ is not included in the list.

**Comment by tobias-liaudat** [comment]
Hi @Jeanne-jw 
Why do you say that _sometimes_ the up limit is not included?

The problem is that the behaviour changes with numpy version or that the 0.9um value should be always included in the list?

**Comment by Jeanne-jw** [comment]
You are right. In `np.arange()`, the interval is half-open as `[start, stop)`, so it's always not included. https://numpy.org/doc/stable/reference/generated/numpy.arange.html
